search-app/docs/api.md

```diff
@@ -458,9 +458,15 @@ __Example__
 
 #### <a name="csv"></a> CSV
 
-The comma separated value (CSV) format is only supported for granules.
+The comma separated value (CSV) format is supported for collections and granules.
 
-__Example__
+__Examples__
+
+```csv
+Data Provider,Short Name,Version,Entry Title,Processing Level,Platforms,Start Time
+PROV1,short,v2,dataset-id,2,platform-1,2015-06-05T20:10:39.166Z
+PROV2,very-short,v1,data-id,1,,2019-08-03T20:11:59.143Z
```

Comment:
Platform is a required field in UMM-C. We will never have a case when there is no platform for a collection. We should fix this example.